app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -81,4 +84,4 @@ function PersonalKeyStep({ value }: PersonalKeyStepProps) {
   );
 }
 
-export default PersonalKeyStep;
+export default withPresenceValidation(PersonalKeyStep, 'personalKey');
```

Comment:
I like the idea of a wrapper function around a component, but I feel like adding it here makes it "hidden"? If I was trying to debug a missing value, I wouldn't check the bottom of the file

Also, do we know that all steps are going to have values? Ex the welcome/agreement steps don't really leave values the same way? But since they're informational it may be OK if they're skipped?

Reply:
Interesting points you raise. I guess an implied alternative is to track and persist step completion by name? i.e. only allow a user to visit personal key if we know they've completed the password confirmation step.